<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>318</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>312</width>
    <height>318</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>312</width>
    <height>318</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator Test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="OutputField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>231</width>
      <height>31</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="digitCount">
     <number>17</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="MemoryStateIndicator">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>295</width>
      <height>239</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>295</width>
      <height>239</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QPushButton" name="SquareButton">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Sqr</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="SquareRootButton">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Sqrt</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="InvertSignButton">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>+/-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="2">
            <widget class="QPushButton" name="DigitButton_3">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="DigitButton_2">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="DigitButton_4">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="DigitButton_5">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="ResultButton">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>=</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="DigitButton_0">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="DigitButton_7">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="DigitButton_9">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>9</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="DigitButton_6">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="DigitButton_8">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>8</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="DigitButton_1">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="SetDotButton">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPushButton" name="ResetButton">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>41</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QPushButton" name="SumButton">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="MinusButton">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="MultiplyButton">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>*</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="DivisionButton">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>41</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>/</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QPushButton" name="ClearMemoryButton">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>41</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>41</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>MC</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QPushButton" name="ShowMemoryButton">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>41</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>MR</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="AddMemoryButton">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>41</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>M+</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="ReplaceMemoryButton">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>41</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>MN</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="MinusMemoryButton">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>41</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>M-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>OutputField</zorder>
   <zorder>MemoryStateIndicator</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>312</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="menu_action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="menu_action_About"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="menu_action_About">
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="toolTip">
    <string>О программе</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="menu_action_Exit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
